CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Feature: Make `Marger` a function instead of a React.Component.
```

Comment:
Non-bloquant : j'ai l'impression que ce serait plus un fixe. On ajoute rien au composant à part un changement de syntaxe ?

Reply:
J'avais opté pour Feature parce qu'on avait précédemment fait ça :

```
## [2.73.0] - 2020-05-29

Features:

- Make `CrowdfundingCard` a function instead of a `PureComponent`.
…
```

Reply:
(mais sinon je suis tout à fait d'accord avec toi)